### ISSUE TITLE
PDFa flavour PDF output rendering - footnote text is too pale

### ISSUE THREAD

**Issue by petervwyatt** [metanorma/metanorma-pdfa#28]
The PDFa flavor PDF output rendering of footnoted text is too pale and fails WCAG contrast rules. HTML appears OK.
For our flavor, please use the default/standard text color (black or near-black?).

See [this ADOC file](https://github.com/petervwyatt/mn-playground/blob/16f1d2aac95ad55e2ab21df6ddff66a2de18f945/testing-spans.adoc?plain=1#L16) in my mn-playground.

<img width="706" height="490" alt="Image" src="https://github.com/user-attachments/assets/b22e6123-dfc1-428b-9a74-e7b795622e0c" />

**Comment by Intelligent2013** [comment]
Done:

<img width="552" height="375" alt="Image" src="https://github.com/user-attachments/assets/7cc96589-06b9-4368-9e7e-d5b827b95893" />

**Comment by petervwyatt** [comment]
Confirmed. Thanks.
